tests/phpunit/tests/functions/__wpCheckAlternateFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::_wp_check_existing_file_names
+ */
+class Tests_Functions__WpCheckAlternateFileNames extends WP_UnitTestCase {
+
+	/**
+	 * @dataProvider data__wp_check_alternate_file_names
+	 *
+	 * @ticket 55192
```

Comment:
```suggestion
	 * @ticket 55199
```
- Corrected ticket number.